### ISSUE TITLE
GPG Signature fails on latest release 5.0.9328

### ISSUE THREAD

**Issue by jonathancross** [BeamMW/beam#1577]
**Bug title**
The signature on the latest release `Checksums.txt` ([beam-5.0.9328](https://github.com/BeamMW/beam/releases/tag/beam-5.0.9328)) seems to be invalid.

**Bug description**
```
gpg -v --verify Checksums.txt.asc 
gpg: assuming signed data in 'Checksums.txt'
gpg: Signature made Thu Jun 18 09:22:14 2020 CEST
gpg:                using RSA key 0x9FEC539DB8E4EA8B
gpg: using subkey 0x9FEC539DB8E4EA8B instead of primary key 0x997EAB9F3DF7F375
gpg: using pgp trust model
gpg: BAD signature from "Beam Privacy <security@beam.mw>" [unknown]
gpg: binary signature, digest algorithm SHA256, key algorithm rsa2048
```
More info on the files I downloaded:
```
→ sha256sum Checksums.txt*
6e9d836210860b663417f224ecc7df462867c283b6216ce6e684193a393eb706  Checksums.txt
a50ecccf0bbfbbdaccfd19e78bdbc27b99ca89328f71bf7464365813c57abb81  Checksums.txt.asc

→ cat Checksums.txt.asc 
-----BEGIN PGP SIGNATURE-----

wsBcBAABCAAQBQJe6xYmCRCf7FOduOTqiwAAdzcIAF+2+IWBOdSgiP4K+DvMQClR
lHLNypd1t0mYx80pzMfazIWQa6Yf2dmpw8NGdg2NsZ7R6zftegQd2CZ433JRTJ/x
plUr97rRoPV6U3JvUSFpbPCAwA+yyeNkH8orIcZ3TbPZoLqP9tBiSovEOPl/1F6P
iGhnz9AyHs0WbVYXii8cIlvcrQi5cCTuf16mXPQebCvZ0HB+hCjs86+NUq2grgEk
Yy2vmGH0Bw92TLmQjmDCq3c8NRMm+nuD854o4+KfGTlYrMDWGreHWF9lwngYpo+L
QOLIyRhrcrndMrNJAbkEMb5pWtQt0bwocmn6kSNshml68e/cjtO/3e3dsRp4Ok4=
=+MIR
-----END PGP SIGNATURE-----
```

**To Reproduce**
1. Download files from github.
2. try to verify with `gpg`

**Current behaviour**
Bad signature suggest files have been tampered with, there was a download error or were improperly signed.
I downloaded files from the release page multiple times via browser and curl -- same result.

**Expected behaviour**
Expect a good signature.

**Checksum file:**
[Checksums.txt](https://github.com/BeamMW/beam/files/5184007/Checksums.txt)

**Comment by gingervik** [comment]
thank you for your opened issue
@jonathancross 
should be fixed for 5.1 release

**Comment by gingervik** [comment]
fixed on 5.1 and 5.2

**Comment by jonathancross** [comment]
@gingervik  Recent releases such as [beam-5.3.10396](https://github.com/BeamMW/beam/releases/tag/beam-5.3.10396) have no signature files.  Please add those.

**Comment by jonathancross** [comment]
@gingervik Nice to see signatures were added for beam-5.3.10396  -- but the new hotfix ([beam-5.3.10410](https://github.com/BeamMW/beam/releases/tag/beam-5.3.10410)) is now missing sigs.  Can this be just a standard practice for all releases please?  It is not wise to ask users to just trust GitHub. Thanks!

**Comment by gingervik** [comment]
Hello @jonathancross 
it takes some time to sign new files. Most of our users download new builds from the website. So we add signatures to Github after doing all other release stuff.

**Comment by jonathancross** [comment]
> Hello @jonathancross it takes some time to sign new files.

Ideally this should be automated with a shell script, etc.
Please let me know if you need help writing such a script.

> Most of our users download new builds from the website. So we add signatures to Github after doing all other release stuff.

It's been 5 months since the last time a `Checksums.txt.asc` file was published in [beam-6.1.12023](https://github.com/BeamMW/beam/releases/tag/beam-6.1.12023).

Signatures should be made available on the website and users should be **_encouraged_** to verify downloads rather than blindly trusting web servers.  So far, Beam is too small to attract these kind of attacks, but if it becomes popular, fake wallet apps will be created and used to steal user funds as is common with Bitcoin, Ethereum, etc.  It is a bit concerning that I have to spell this out.
